release-notes/opensearch-knn.release-notes-2.2.0.0.md

```diff
@@ -0,0 +1,31 @@
+## Version 2.2.0.0 Release Notes
+
+Compatible with OpenSearch 2.2.0
+
+### Features
+* Lucene Based k-NN search support([#486](https://github.com/opensearch-project/k-NN/pull/486/commits))
```

Comment:
I main PR there are child PRs listed [485](https://github.com/opensearch-project/k-NN/pull/485), do we need link to commits?